content/docs/user-guide/programming/gems/code-gems.md

```diff
@@ -43,3 +43,23 @@ The `<Gem>EditorModule` class contains the logic for using the Gem in the Editor
 ### `<Gem>EditorSystemComponent.cpp`, `<Gem>EditorSystemComponent.h`
 
 The `<Gem>EditorSystemComponent` class is the system component class for managing the Editor module. You can use this class to program Editor functionality. For example, if your Gem implements a component or a window that appears in the O3DE Editor.
+
+## Building a Gem
+
+After a Gem is registered to an engine, project, or the global `o3de_manifest.json`, you can build the gem using the same command line tool that you use to [build the engine and projects](/docs/user-guide/build/).  Gems that have been added to a project will be built automatically when the project builds.  While developing a Gem, however, you may want to build it individually to avoid rebuilding the entire project.
+
+In the directory that is appropriate for your [engine installation](/docs/welcome-guide/setup/setup-from-github/building-windows/#build-the-engine), use the `cmake --build` command and specify the build solution and configuration.  Use the `--target` flag to specify the Gem modules that you want to build, `<Gem Name>` for the main Gem module or `<Gem Name>.Editor` for Editor modules.  
+
+The following example shows the command to build _both_ the Editor and main Gem modules for a Gem named `MyGem`:
```

Comment:
@lumberyard-employee-dm @FiniteStateGit When would we want to build only the Editor modules, or only the Gem modules, or both?

Reply:
As @FiniteStateGit says depending on what targets are available in the gem, that would determine what to build.

For example the DefaultGem template contains both a game and editor target. 
The user could then choose to remove the game target from the Gem's CMakeLists.txt. In which case, the gem becomes a tools only gem and only the editor module needs to be built.
Alternatively the user could then choose to remove the editor target from the Gem's CMakeLists.txt. In this situation, the gem is a runtime(game/simulation) only gem and only the main Gem module target would need to be built.

Reply:
Thanks for clarifying! That's exactly the bit I think was missing in my understanding. Might be helpful for folks who are new to Gem development to understand that the nature of a code Gem can be either for Editor Mode or Game Mode, and as a result, there are different modules to build. Out of scope for this PR, but info I think would be helpful in the Gem section.